### ISSUE TITLE
Enable without use of button

### ISSUE THREAD

**Issue by nosehair** [bopen/leaflet-area-selection#6]
How can I activate the draw feature manually?

So something like
```
const areaSelection = new DrawAreaSelection({ withButton: false})
areaSelection.addTo(map)

areaSelection.activate()
```

I'd like to use my own button and place it outside of the map area.

**Comment by keul** [comment]
@nosehair not possible right now, but see https://github.com/bopen/leaflet-area-selection/pull/5

**Comment by keul** [comment]
Closed by #5

**Comment by keul** [comment]
Example: https://github.com/bopen/leaflet-area-selection/blob/aeaec30f0500c3e5064e43856d403abfee8da872/example/src/index.js#L77
